### ISSUE TITLE
./install-playstore.sh: line 155: /etc/systemd/system/anbox-container-manager.service.d/override.conf: No such file or directory

### ISSUE THREAD

**Issue by 4joey1** [geeks-r-us/anbox-playstore-installer#66]
I am getting error below when I run the ./install-playstore.sh script

./install-playstore.sh: line 155: /etc/systemd/system/anbox-container-manager.service.d/override.conf: No such file or directory

**Desktop (please complete the following information):**
 - OS: [e.g. Debian]
 - Version [e.g. bullseye/sid]

**Additional context**

**Comment by 4joey1** [comment]
When I got error below

`install-playstore.sh: line 155: /etc/systemd/system/anbox-container-manager.service.d/override.conf: No such file or directory`

I followed the thread on https://github.com/geeks-r-us/anbox-playstore-installer/issues/54  and changed my line 155 from` $SUDO cat >/etc/systemd/system/snap.anbox.container-manager.service/override.conf<<EOF` to  `$SUDO cat >/lib/systemd/system/anbox-container-manager.service/override.conf<<EOF` and ran the script again but now got error below

`./install-playstore.sh: line 155: /lib/systemd/system/anbox-container-manager.service/override.conf: Not a directory`

I removed the override.conf line and when I ran the script again I now got the following message

```
/var/lib/anbox/rootfs-overlay
Overlay no enabled ! Please check error messages!
```
Please confirm if I have to remove the override.conf line and how doe I finally resolve my issue?

**Comment by 13banda** [comment]
just make sure  anbox install using snap and remove other installation
`which anybox` to find installation folder . make sure it is under /snap or /var/snap

**Comment by Anil-v29** [comment]
Having the same issue!

**Comment by Anil-v29** [comment]
> just make sure anbox install using snap and remove other installation
> `which anybox` to find installation folder . make sure it is under /snap or /var/snap

Can you elaborate? I don't understand

**Comment by happyeggchen** [comment]
ok,try this guys
````
sudo sed -i 's/ExecStart="/ExecStart=\/usr\/bin\/anbox container-manager --daemon --privileged --data-path=\/var\/lib\/anbox --use-rootfs-overlay/g' /lib/systemd/system/anbox-container-manager.service
```
This will force it using rootfs-overlay
But you have to install anbox in your system

**Comment by happyeggchen** [comment]
Or you can do it manual,it's about exec in its systemd service

**Comment by tzargoth** [comment]
On my Ubuntu 18.04 system, ```which anbox``` returns '/usr/bin/anbox', which is not expected by the install script.  This results in the error mentioned by the OP.  However, "/var/snap/anbox/common" exists, so I added below to line 134 to include check for alternate anbox executable location:
```
 || [ ${ANBOX} = "/usr/bin/anbox" ]
```
With this, my installation has proceeded successfully.

**Comment by Zeeroni** [comment]
On Debian I had to run:
`sudo mkdir /etc/systemd/system/anbox-container-manager.service.d`
before running the script

This is where systemd looks for the override.conf file for /lib/systemd/system/anbox-container-manager.service. Since that file has no overrides by default, the directory /etc/systemd/system/anbox-container-manager.service.d/ doesn't exist by default on Debian systems. This script needs some extra code to create it.

**Comment by leijie2015** [comment]
> On Debian I had to run: `sudo mkdir /etc/systemd/system/anbox-container-manager.service.d` before running the script
> 
> This is where systemd looks for the override.conf file for /lib/systemd/system/anbox-container-manager.service. Since that file has no overrides by default, the directory /etc/systemd/system/anbox-container-manager.service.d/ doesn't exist by default on Debian systems. This script needs some extra code to create it.

This is a correct answer, and it works for me!

**Comment by geeks-r-us** [comment]
should be fixed with #70
